structures.h

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "structures.c"

```

Comment:
delete #include "structures.c"

here is the place for your functions implementatiions

Reply:
Tried deleting, main.c isn't working without #include "structures.c" in this header.

Reply:
becouse U compile it wrong

Reply:
How are U create project? Do U have Makefile?

Reply:
read LectC.pdf - last chapter

Reply:
Codeblocks created project for me, having "project file" in folder